### PR TITLE
docs: fix some minor typos

### PR DESCRIPTION
**Pull request by johannesschobel**
This PR addresses some typos i found when reading the docs.

### CODE REVIEW

File: content/recipes/sql-typeorm.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### SQL (TypeORM)
 
-##### This chapter applies only to TypeScript
+#### This chapter applies only to TypeScript
```
Comment (kamilmysliwiec) [inline content/recipes/sql-typeorm.md]: This is intentional :)

File: content/recipes/sql-sequelize.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ### SQL (Sequelize)
 
-##### This chapter applies only to TypeScript
+#### This chapter applies only to TypeScript
```
Comment (kamilmysliwiec) [inline content/recipes/sql-sequelize.md]: This is intentional (styling + margins) :)

File: content/recipes/sql-typeorm.md (hunk 1 above)
Comment (johannesschobel) [reply content/recipes/sql-typeorm.md]: undo ;)

File: content/recipes/sql-sequelize.md (hunk 2 above)
Comment (johannesschobel) [reply content/recipes/sql-sequelize.md]: undo ;)

**Pull request merged**